src/NzbDrone.Core/Indexers/Definitions/Orpheus.cs

```diff
@@ -50,5 +51,32 @@ protected override IndexerCapabilities SetCapabilities()
 
             return caps;
         }
+
+        public override IParseIndexerResponse GetParser()
+        {
+            return new OrpheusParser(Settings, Capabilities);
+        }
+    }
+
+    public class OrpheusParser : GazelleParser
+    {
+        public OrpheusParser(GazelleSettings settings, IndexerCapabilities capabilities)
+            : base(settings, capabilities)
+        {
+        }
+
+        protected override string GetDownloadUrl(int torrentId)
+        {
+            var url = new HttpUri(_settings.BaseUrl)
+                .CombinePath("/torrents.php")
+                .AddQueryParam("action", "download")
+                .AddQueryParam("useToken", _settings.UseFreeleechToken ? "1" : "0")
```

Comment:
Remove this, then add it as a separate statement only if `_settings.UseFreeleechToken` is true